catalog/.prettierignore

```diff
@@ -2,3 +2,5 @@ build/
 node_modules/
 internals/generators/
 package-lock.json
+*.generated.ts
```

Comment:
that's not required, since generated files are run through prettier when running the generation script